src/pages/jobSearch/JobSearchPage.tsx

```
@@ -32,14 +50,44 @@ const JobSearchPage = () => {
-        <p className={'text-purple-500 font-T03-B'}>{'8'}개</p>의 일자리가 구인
-        중이에요
+        <p className="text-purple-500 font-T03-B">{data?.total ?? 0}개</p>의
+        일자리가 구인 중이에요
       </div>
       <div className="container mx-auto mb-6 px-4 pt-8">
-        <div className="grid grid-cols-1 gap-4 md:grid-cols-2 lg:grid-cols-3">
-          {RecruitDummy.map((item: RecruitItem) => (
-            <RecruitCard key={item.id} item={item} />
-          ))}
+        <div className="grid grid-cols-1 items-stretch gap-4 md:grid-cols-2 lg:grid-cols-3">
+          {jobs.map((item: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)